lib/titanium.js

```diff
@@ -412,7 +412,8 @@ function run(locale) {
 			if (supported === false) {
 				logger.banner();
 				logger.error(__('Titanium SDK %s is incompatible with Node.js %s', sdk.name, process.version) + '\n');
-				logger.log(__('You will need to install Node.js %s in order to use this version of the Titanium SDK.', 'v' + appc.version.parseMax(sdk.packageJson.vendorDependencies.node)));
+				logger.log(__('You will need to install Node.js %s or higher in order to use this version of the Titanium SDK. However, we recommend to install maximum supported version Node.js %s.', 'v' + appc.version.parseMin(sdk.packageJson.vendorDependencies.node),'v' + appc.version.parseMax(sdk.packageJson.vendorDependencies.node)));
```

Comment:
I don't agree with this. We should only support Node.js versions that are supported by Node.js. Node.js 5.x is no longer maintained, thus we should not support it. That means that Titanium SDK's package.json should list the supported Node.js versions as `4.x || 6.x`. Your verbiage doesn't account for a gap in version.

Titanium SDK 5.4.0 only supports 0.10, 0.12, and 4.x. It will not work with Node.js 6. I'm not sure if it works with Node.js 5.x, but it doesn't matter because we don't test Node.js 5.x and thus it's not supported.

Titanium SDK 5.5.0 only supports 0.10, 0.12, and 4.x. I updated node-ios-device so that it will work with Node.js 5 and 6, however we don't test Node.js 5 and 6 and thus we don't support Node.js 5 and 6.

Titanium SDK 6.0.0 will only support Node.js 4.x and 6.x. It may work with 5.x or <4.0, but we won't test/support it.

I recommend we treat the Node.js version similarly to how we handle Xcode and Android SDK versions by errorring out if the version is less than the minimum supported version and showing a warning if the version is greater than the max version. See the verbiage for Xcode: https://github.com/appcelerator/titanium_mobile/blob/master/iphone/cli/lib/info.js#L67-L74.